apps/cli/src/command/lint.command.ts

```diff
@@ -57,13 +57,15 @@ export const LintTask = (): ListrTask<{ local: ADCSDK.Configuration }> => ({
 
 export const LintCommand = new BaseCommand('lint')
   .description(
-    'Check provided configuration files, local execution only, ensuring inputs meet ADC requirements',
+    'Lint the local configuration file(s) to ensure it meets ADC requirements.',
   )
+	.summary('lint the local configuration')
   .option(
     '-f, --file <file-path...>',
-    'The files you want to synchronize, can be set more than one.',
+    'files to synchronize',
```

Comment:
Should it be `file(s) to lint`?

Reply:
When there are more than one file, it is `-f file1.yaml -f file2.yaml` right?

Since it is not `-f file1.yaml file2.yaml`, maybe it should be `File to xxxx. Can be set more than once.`?

Reply:
It should be `file`. Thanks for pointing it out. "Can be set more than once" can be inferred from the example? Is it necessary to specify it here?

Reply:
> "Can be set more than once" can be inferred from the example?

Sounds good.